src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -49,6 +49,10 @@ namespace Maestro.Web
 {
     public partial class Startup
     {
+        // TODO: Remove once the repo in this list is ready to onboard to yaml publishing.
```

Comment:
Nit: I'd reference the tracking bug here in case someone is curious about this workaround